content/news/posts/2023/a-new-organization-for-the-open-source-project.md

```diff
@@ -0,0 +1,152 @@
+
+* Admittance in a Council or delegate group is mainly based on good will, mutual trust, reputation, engagement, and convergence toward common goals. We work in small groups and we trust each other to work in the project's best interest.
+* We optimize for efficacy and not for thoroughness. We set up lean processes, only when such processes are a necessity. Not everything needs to be written in stone for an action to take place.
+* We use [lazy consensus](https://community.apache.org/committers/lazyConsensus.html) to accelerate decision-making, notably by specifically stating short (but reasonable) timeframes for providing feedback on a proposal.
+* We cherish the humility to ask when we don't know, recognize when we are wrong, accept when we have made a mistake, learn from it, and improve.
+* Less is more. We strive not to overthink things!
+
+## Initial composition of Project Councils
+
+The initial composition of the project's Councils is largely based on the established roles from our previous structure. Current maintainers (including scoped maintainers) will automatically join the Technical Council. The rest of the Councils will be initially populated with the people currently performing similar roles.
+
+As we roll out the new structure out and Councils begin to function, new members may be invited, and inactive members may be removed.
+
+**Leadership Council**
+
+* Pablo Borowicz (@eternoendless) **– Project Lead**
+* Fabien Vallon (@fabienvallon)
+* Mathieu Ferment (@matks)
+* Robin Fischer (@Robin-Fischer-PS)
+
+**Technical Council**
+
+* Jonathan Lelièvre (@jolelievre) **– Lead**
+* Daniel Hlavacek (@Hlavtox)
+* Ibrahima Sow (@sowbiba)
+* Krystian Podemski (@kpodemski)
+* Mathieu Ferment (@matks)
+* Matthieu Rolland (@matthieu-rolland)
+* Pablo Borowicz (@eternoendless)
+* Pululu Kinanga André (@PululuK)
+* Thomas Baccelli (@atomiix)
+* Valentin Szczupak (@NeOMakinG)
+
+**Product Council**
+
+* Fabien Vallon (@fabienvallon) **– Lead**
+* Mateus Shirlaw (@MatShir)
+* Rosa Benouamer (@RosaBenouamer)
+* Zoé Finders (@Zoefinders)
+
+**Quality Council**
+
+* Robin Fischer (@Robin-Fischer-PS) **– Lead**
+* Boubker Bribri (@boubkerbribri)
+
+## Advantages of the new organization
+
+We think that this structure will provide many advantages:
+
+It will **facilitate decision-making**, thanks to a clearly-defined project structure, including the people responsible for each domain, that clears out any misunderstanding as to who is responsible for what decision (and especially who has the "final-final-final" word on it).
+
+It will **enable more people to participate in the project,** by opening decision-making roles that were previously closed to community members, including the creation of new lower-stakes roles that are easier to join to.
```

Comment:
```suggestion
It will **enable more people to participate in the project,** by opening decision-making roles that were previously closed to community members, including the creation of new lower-stakes roles that are easier to join.
```